docs/release_process.md

```diff
@@ -265,11 +260,21 @@ The general rule is that the PR Titles will compose the body of the changelog.
 
 ### Tagging
 
-- Create and push a **vX.Y.Z** tag to the GitHub repo.
+This set of commands describes how to push the release tag.
 ```
-git tag vX.Y.Z stabilization-vX.Y.Z
-git push --tags
+release=v0.1.99
+remote=origin
+
+# get up-to-date branches and tags from github
+git fetch --tags $remote
+
+# create a new tag based on latest stabilization
+git tag $release $remote/stabilization
+
+# push the tag
+git push $remote $release
```

Comment:
The reason why I had tag creation + push + stable update in one code block is to ensure that everything is done at the same time, in the same "sorta atomic" time.

If you split it up, it might appear that one can wait some time before ie. fetching `stabilization` and updating `stable`, which is incorrect - the `stable` update **relies** on the local copy of `stabilization` being up to date.

So either duplicate the `git fetch --tags $remote` in the second section further below, or merge the two sections.